cmd/explaintest/r/select.result

```diff
@@ -470,3 +470,16 @@ HashLeftJoin_10	7984.01	root	semi join, inner:TableReader_16, equal:[eq(test.t1.
 └─TableReader_16	9980.01	root	data:Selection_15
   └─Selection_15	9980.01	cop	not(isnull(test.t2.a)), not(isnull(test.t2.b))
     └─TableScan_14	10000.00	cop	table:t2, range:[-inf,+inf], keep order:false, stats:pseudo
+ drop table t;
```

Comment:
You can use `./run-tests.sh -r select` to generate the result, it has a leading blank space now.

Reply:
Thanks